src/rest/routes/sync/index.ts

```diff
@@ -0,0 +1,74 @@
+import { Request, Response } from "express";
+import { ParamsDictionary } from "express-serve-static-core";
+import { errorWrapper } from "../../helper";
+import { Subscription } from "models/subscription";
+import { findOrStartSync } from "~/src/sync/sync-utils";
+import { determineSyncTypeAndTargetTasks } from "~/src/util/github-sync-helper";
+import { BaseLocals } from "..";
+import { RestApiError } from "~/src/config/errors";
+import { RestSyncReqBody } from "~/src/rest-interfaces";
+
+
+const restSyncPost = async (
+	req: Request<ParamsDictionary, unknown, RestSyncReqBody>,
+	res: Response<unknown, BaseLocals>
+) => {
+	//TODO: We are yet to handle enterprise backfill
+	const cloudOrUUID = req.params.cloudOrUUID;
+	const gheUUID = cloudOrUUID === "cloud" ? undefined : req.params.cloudOrUUID;
+	let gitHubAppId : number | undefined = undefined;
+	if (gheUUID) {
+		gitHubAppId = parseFloat(gheUUID);
```

Comment:
`UUID` and `githubAppId` are 2 completely different things.
We have `UUID` in the frontend, which we use to identify the exact GHE app, so in order to get the `githubAppId`, you will get that from the table `GitHubServerApp`
` await GitHubServerApp.getForUuidAndInstallationId(UUID, res.locals.installation.id)`